transport/nio/src/main/java/org/kaazing/gateway/transport/nio/internal/NioSocketAcceptor.java

```diff
@@ -123,12 +123,12 @@ public NioWorker call() throws Exception {
 
     private final AtomicReference<DistributedNioWorkerPool> currentWorkerPool = new AtomicReference<>();
 
-    NioSocketAcceptor(Properties configuration, TcpExtensionFactory extensionFactory) {
+    public NioSocketAcceptor(Properties configuration, TcpExtensionFactory extensionFactory) {
         super(configuration, LoggerFactory.getLogger(LOGGER_NAME));
         this.extensionFactory = extensionFactory;
     }
 
-    public NioSocketAcceptor(Properties configuration) {
+    NioSocketAcceptor(Properties configuration) {
         super(configuration, LoggerFactory.getLogger(LOGGER_NAME));
```

Comment:
just call the above constructor by doing
this(configuration, TcpExtensionFactory.newInstance())